root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -5159,6 +5159,20 @@ limited_link_type_combinations: [
             expected_clean_url: 'https://www.uta-net.com/artist/9208/',
        only_valid_entity_types: ['artist'],
   },
+  {
+                     input_url: 'https://www.uta-net.com/composer/30/7/',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'lyrics',
+            expected_clean_url: 'https://www.uta-net.com/composer/30/',
+       only_valid_entity_types: ['artist'],
+  },
+  {
+                     input_url: 'https://www.uta-net.com/lyricist/21/5/',
```

Comment:
For a bit of variation, based on real top link:
```suggestion
                     input_url: 'https://www.uta-net.com/lyricist/21/#top',
```